api/src/services/inscription-goal.ts

```diff
@@ -48,15 +48,18 @@ export const getCompletionObjectifDepartement = async (department: string, cohor
 };
 
 // Vérification des objectifs pour un département et la région associée
-export const getCompletionObjectifs = async (department: string, cohort: string) => {
-  const completionObjectifDepartement = await getCompletionObjectifDepartement(department, cohort);
-  const completionObjectifRegion = await getCompletionObjectifRegion(department2region[department], cohort);
+export const getCompletionObjectifs = async (department: string, cohortName: string, objectifLevel: string) => {
```

Comment:
```suggestion
// TODO: passer l'object "cohort" directement
export const getCompletionObjectifs = async (department: string, cohortName: string, cohortObjectifLevel: string) => {
```
a minima, renommer `objectifLevel` en `cohortObjectifLevel` pour que ce ça soit plus clair à la lecture

Reply:
J'ai mis un objet du coup